src/components/molecules/NavSearchBar/NavSearchList.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { Key } from "react";
+import { FixedSizeList } from "react-window";
+
+import "./NavSearchList.scss";
+
+import {
+  NavSearchResultProps,
+  NavSearchResult,
+} from "components/molecules/NavSearchBar/NavSearchResult";
+
+// @debt refactor this into .scss or find more ergonomic component with same desired functionality as FixedSizeList
+// values are used here due to quirks of the FixedSizeList component
+// NOTE: keep in sync with the .scss files mentioned in comments
+const propsFor = (length: number) => ({
```

Comment:
```suggestion
const getFixedSizeListProps = (length: number) => ({
```

Reply:
Why name a non getter functions with get? Shouldn't be more `calculateFixedSizeListProps`?